README.md

```diff
@@ -12,7 +12,7 @@ The configuration of the Kafka Datasource has the following JSON format:
 
 ```
 {
- "bootstrapServers": Array of Strings "host:port",
+ "bootstrapServers": Array of Strings,
```

Comment:
Was this change necessary? Unlike other places, Kafka _requires_ the `:port` part of it.

Reply:
I'll change it back